File: backend/backend-app/lambda-deployment/src/constants/municipios.py
MUNICIPIOS_POR_SUBREGION = {
    "Abades": ["Providencia", "Samaniego", "Santacruz"],
    "Centro": ["Chachagüí", "La Florida", "Nariño", "Pasto", "Tangua", "Yacuanquer"],
    "Cordillera": ["Cumbitara", "El Rosario", "Leiva", "Policarpa", "Taminango"],
    "Exprovincia de Obando": [
        "Aldana", "Contadero", "Córdoba", "Cuaspud", "Cumbal", "Funes", "Guachucal",
        "Gualmatán", "Iles", "Ipiales", "Potosí", "Puerres", "Pupiales"
    ],
    "Guambuyaco": ["El Peñol", "El Tambo", "La Llanada", "Los Andes"],
    "Juanambú": ["Arboleda", "Buesaco", "La Unión", "San Lorenzo", "San Pedro de Cartago"],
    "Occidente": ["Ancuya", "Consacá", "Linares", "Sandoná"],
    "Pacífico Sur": ["Francisco Pizarro", "San Andrés de Tumaco"],
    "Pie de Monte Costero": ["Mallama", "Ricaurte"],
    "Río Mayo": ["Albán", "Belén", "Colón", "El Tablón de Gómez", "La Cruz", "San Bernardo", "San Pablo"],
    "Sabana": ["Guaitarilla", "Imués", "Ospina", "Sapuyes", "Túquerres"],
    "Sanquianga": ["El Charco", "La Tola", "Mosquera", "Olaya Herrera", "Santa Bárbara"],
    "Telembí": ["Barbacoas", "Magüí", "Roberto Payán"],
}

LISTA_MUNICIPIOS = sorted({m for lista in MUNICIPIOS_POR_SUBREGION.values() for m in lista})

# Cupos por municipio (valores de los TDR)
MUNICIPIO_A_CUPO = {
    'Providencia': 3, 'Samaniego': 12, 'Santacruz': 5,
    'Chachagüí': 7, 'La Florida': 4, 'Nariño': 3, 'Pasto': 175, 'Tangua': 6, 'Yacuanquer': 5,
    'Cumbitara': 4, 'El Rosario': 5, 'Leiva': 4, 'Policarpa': 4, 'Taminango': 8,
    'Aldana': 3, 'Contadero': 3, 'Córdoba': 7, 'Cuaspud': 4, 'Cumbal': 17, 'Funes': 3, 'Guachucal': 9,
    'Gualmatán': 3, 'Iles': 3, 'Ipiales': 52, 'Potosí': 4, 'Puerres': 4, 'Pupiales': 8,
    'El Peñol': 4, 'El Tambo': 6, 'La Llanada': 3, 'Los Andes': 4,
    'Arboleda': 4, 'Buesaco': 11, 'La Unión': 14, 'San Lorenzo': 8, 'San Pedro de Cartago': 3,
    'Ancuya': 4, 'Consacá': 6, 'Linares': 4, 'Sandoná': 9,
    'Francisco Pizarro': 6, 'San Andrés de Tumaco': 115,
    'Mallama': 4, 'Ricaurte': 9,
    'Albán': 4, 'Belén': 3, 'Colón': 4, 'El Tablón de Gómez': 6, 'La Cruz': 8, 'San Bernardo': 4, 'San Pablo': 7,
    'Guaitarilla': 6, 'Imués': 4, 'Ospina': 4, 'Sapuyes': 6, 'Túquerres': 15,
    'El Charco': 10, 'La Tola': 3, 'Mosquera': 6, 'Olaya Herrera': 11, 'Santa Bárbara': 6,
    'Barbacoas': 25, 'Magüí': 11, 'Roberto Payán': 6,
}

File: frontend/frontend-app/src/constants/municipios.js
export const MUNICIPIOS_POR_SUBREGION = {
  Abades: [
    { nombre: 'Providencia', cupo: 3 },
    { nombre: 'Samaniego', cupo: 12 },
    { nombre: 'Santacruz', cupo: 5 },
  ],
  Centro: [
    { nombre: 'Chachagüí', cupo: 7 },
    { nombre: 'La Florida', cupo: 4 },
    { nombre: 'Nariño', cupo: 3 },
    { nombre: 'Pasto', cupo: 175 },
    { nombre: 'Tangua', cupo: 6 },
    { nombre: 'Yacuanquer', cupo: 5 },
  ],
  Cordillera: [
    { nombre: 'Cumbitara', cupo: 4 },
    { nombre: 'El Rosario', cupo: 5 },
    { nombre: 'Leiva', cupo: 4 },
    { nombre: 'Policarpa', cupo: 4 },
    { nombre: 'Taminango', cupo: 8 },
  ],
  'Exprovincia de Obando': [
    { nombre: 'Aldana', cupo: 3 },
    { nombre: 'Contadero', cupo: 3 },
    { nombre: 'Córdoba', cupo: 7 },
    { nombre: 'Cuaspud', cupo: 4 },
    { nombre: 'Cumbal', cupo: 17 },
    { nombre: 'Funes', cupo: 3 },
    { nombre: 'Guachucal', cupo: 9 },
    { nombre: 'Gualmatán', cupo: 3 },
    { nombre: 'Iles', cupo: 3 },
    { nombre: 'Ipiales', cupo: 52 },
    { nombre: 'Potosí', cupo: 4 },
    { nombre: 'Puerres', cupo: 4 },
    { nombre: 'Pupiales', cupo: 8 },
  ],
  Guambuyaco: [
    { nombre: 'El Peñol', cupo: 4 },
    { nombre: 'El Tambo', cupo: 6 },
    { nombre: 'La Llanada', cupo: 3 },
    { nombre: 'Los Andes', cupo: 4 },
  ],
  Juanambú: [
    { nombre: 'Arboleda', cupo: 4 },
    { nombre: 'Buesaco', cupo: 11 },
    { nombre: 'La Unión', cupo: 14 },
    { nombre: 'San Lorenzo', cupo: 8 },
    { nombre: 'San Pedro de Cartago', cupo: 3 },
  ],
  Occidente: [
    { nombre: 'Ancuya', cupo: 4 },
    { nombre: 'Consacá', cupo: 6 },
    { nombre: 'Linares', cupo: 4 },
    { nombre: 'Sandoná', cupo: 9 },
  ],
  'Pacífico Sur': [
    { nombre: 'Francisco Pizarro', cupo: 6 },
    { nombre: 'San Andrés de Tumaco', cupo: 115 },
  ],
  'Pie de Monte Costero': [
    { nombre: 'Mallama', cupo: 4 },
    { nombre: 'Ricaurte', cupo: 9 },
  ],
  'Río Mayo': [
    { nombre: 'Albán', cupo: 4 },
    { nombre: 'Belén', cupo: 3 },
    { nombre: 'Colón', cupo: 4 },
    { nombre: 'El Tablón de Gómez', cupo: 6 },
    { nombre: 'La Cruz', cupo: 8 },
    { nombre: 'San Bernardo', cupo: 4 },
    { nombre: 'San Pablo', cupo: 7 },
  ],
  Sabana: [
    { nombre: 'Guaitarilla', cupo: 6 },
    { nombre: 'Imués', cupo: 4 },
    { nombre: 'Ospina', cupo: 4 },
    { nombre: 'Sapuyes', cupo: 6 },
    { nombre: 'Túquerres', cupo: 15 },
  ],
  Sanquianga: [
    { nombre: 'El Charco', cupo: 10 },
    { nombre: 'La Tola', cupo: 3 },
    { nombre: 'Mosquera', cupo: 6 },
    { nombre: 'Olaya Herrera', cupo: 11 },
    { nombre: 'Santa Bárbara', cupo: 6 },
  ],
  Telembí: [
    { nombre: 'Barbacoas', cupo: 25 },
    { nombre: 'Magüí', cupo: 11 },
    { nombre: 'Roberto Payán', cupo: 6 },
  ],
};

export const LISTA_MUNICIPIOS = Object.values(MUNICIPIOS_POR_SUBREGION).flat().map(m => m.nombre);


